rotary/rules/annotation.smk

```diff
@@ -376,11 +381,11 @@ rule symlink_logs:
 rule summarize_annotation:
     input:
         "{sample}/annotation/dfast/{sample}_genome.fna",
-        "{sample}/annotation/eggnog/{sample}.emapper.annotations",
-        "{sample}/annotation/gtdbtk/{sample}_gtdbtk.summary.tsv",
-        "{sample}/annotation/checkm/",
-        expand("{{sample}}/annotation/coverage/{{sample}}_{type}_coverage.tsv",
-            type=["short_read", "long_read"] if POLISH_WITH_SHORT_READS == True else ["long_read"]),
+        "{sample}/annotation/eggnog/{sample}.emapper.annotations" if ANNOTATION_MAP.eggnog else [],
+        "{sample}/annotation/gtdbtk/{sample}_gtdbtk.summary.tsv" if ANNOTATION_MAP.gtdbtk else [],
+        "{sample}/annotation/checkm/" if ANNOTATION_MAP.checkm2 else [],
+        "{sample}/annotation/coverage/{sample}_short_read_coverage.tsv" if ANNOTATION_MAP.coverage else [],
+        "{sample}/annotation/coverage/{sample}_long_read_coverage.tsv" if POLISH_WITH_SHORT_READS and ANNOTATION_MAP.coverage else [],
```

Comment:
I think `POLISH_WITH_SHORT_READS` should be in the conditional for the `{sample}_short_read_coverage` (rather than the long read coverage) file, correct?